packages/contracts-rfq/contracts/FastBridgeV2.sol

```diff
@@ -183,31 +182,34 @@ contract FastBridgeV2 is Admin, IFastBridgeV2 {
     function prove(bytes memory request, bytes32 destTxHash, address relayer) public onlyRole(RELAYER_ROLE) {
         bytes32 transactionId = keccak256(request);
         // update bridge tx status given proof provided
-        if (bridgeStatuses[transactionId] != BridgeStatus.REQUESTED) revert StatusIncorrect();
-        bridgeStatuses[transactionId] = BridgeStatus.RELAYER_PROVED;
-        bridgeProofs[transactionId] = BridgeProof({timestamp: uint96(block.timestamp), relayer: relayer}); // overflow ok
+        if (bridgeTxDetails[transactionId].status != BridgeStatus.REQUESTED) revert StatusIncorrect();
+        BridgeTxDetails storage _thisBridgeTxDetails = bridgeTxDetails[transactionId];
+        _thisBridgeTxDetails.status = BridgeStatus.RELAYER_PROVED;
+        _thisBridgeTxDetails.proofBlockTimestamp = uint40(block.timestamp);
+        _thisBridgeTxDetails.proofBlockNumber = uint48(block.number);
+        _thisBridgeTxDetails.proofRelayer = relayer;
```

Comment:
Why not just `bridgeTxDetails[transactionId].status = BridgeStatus.RELAYER_PROVED` and so on here?